aries-site/src/examples/templates/list-views/ListScreenExample.js

```diff
@@ -167,7 +167,7 @@ const PageHeaderExample = ({ title }) => (
 
 const AppHeaderExample = () => (
   <Header pad={{ vertical: 'small' }}>
-    <Button>
+    <Button plain>
```

Comment:
For the <Text> instances on lines 180 & 183, I think we need to add color="text" -- right now coming across as text-strong